### PR TITLE
36 sidebar functionality

### PR DESCRIPTION
**Pull request by valselviz**
image

Adding the functionallity for the buttons of "Last month" "last week", "coming soon" and "best of the year"

https://trello.com/c/ayGDIBpt/36-side-bar-functionality-optional

### CODE REVIEW

**Comment by elios11** [comment]
Wow, really nice :)

File: src/components/SidebarNav/SidebarNav.jsx [hunk 1]
```diff
@@ -1,14 +1,66 @@
...
+            setResult(results);
+        }
+    }, [results]);
+
+    const formatDate = (date) => {
+        return date.toISOString().split('T')[0];
+    };
+
+    const thisWeekSearch = () => {
+        const currentDate = new Date();
```
Comment (elios11) [inline src/components/SidebarNav/SidebarNav.jsx:33]: Why is this being defined but not used?

**Review by amililime — APPROVED** [review]

**Review by amililime — APPROVED** [review]

**Review by elios11 — APPROVED** [review]

File: src/components/SidebarNav/SidebarNav.jsx (hunk 1 above)
Comment (valselviz) [reply src/components/SidebarNav/SidebarNav.jsx:33]: maybe this is an old comment but this function is used in line 74

Comment (elios11) [reply src/components/SidebarNav/SidebarNav.jsx:33]: Not the function, the line

Comment (elios11) [reply src/components/SidebarNav/SidebarNav.jsx:33]: I meant the variable in line 32, but I definitely overlooked the template literal, mb :)

**Pull request merged**